core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/listeners/SpanChatModelListener.java

```diff
@@ -78,6 +82,7 @@ public void onResponse(ChatModelResponseContext responseContext) {
                     span.setAttribute("gen_ai.client.estimated_cost", costEstimate.toString());
                 }
             }
+            chatModelSpanContributor.onResponse(responseContext, span);
```

Comment:
This is not a safe operation. We don't control the code being executed there, meaning it can throw an exception and the span will never close.
This should be wrapped on a try/catch...
The whole span processing should be on a try/catch